### PR TITLE
feat: ease multi-database setup by managing mongoose connection like sequelize

### PR DESCRIPTION
**Pull request by jeffladiray**
## Pull Request checklist:

- [ ] Write an explicit title for the Pull Request, following [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- [ ] Create automatic tests
- [ ] No automatic tests failures
- [ ] Test manually the implemented changes
- [ ] Review my own code (indentation, syntax, style, simplicity, readability)
- [ ] Wonder if you can improve the existing code

### CODE REVIEW

File: test-expected/sequelize/dumper-output/index.expected.js [hunk 1]
```diff
@@ -2,50 +2,50 @@ const fs = require('fs');
...
+    .filter((file) => file.indexOf('.') !== 0 && file !== 'index.js')
+    .forEach((file) => {
+      try {
+        const model = connection.import(path.join(__dirname, file));
+        db[model.name] = model;
+      } catch (error) {
+        console.error('Model creation error: ' + error);
+      }
+    });
 });
```
Comment (arnaud-moncel) [inline test-expected/sequelize/dumper-output/index.expected.js:47]: You should put here association 
```suggestion
Object.keys(db).forEach((modelName) => {	
  if ('associate' in db[modelName]) {	
    db[modelName].associate(db);	
  }
}
```

File: test-expected/sequelize/dumper-output/index.expected.js [hunk 2]
```diff
@@ -2,50 +2,50 @@ const fs = require('fs');
...
+  const connection = new Sequelize(databaseInfo.connection.url, databaseOptions);
+  connections[databaseInfo.name] = connection;
+
+  const modelsDir = databaseInfo.modelsDir || databaseInfo.name;
+  fs
+    .readdirSync(path.join(__dirname, modelsDir))
+    .filter((file) => file.indexOf('.') !== 0 && file !== 'index.js')
+    .forEach((file) => {
+      try {
+        const model = connection.import(path.join(__dirname, file));
```
Comment (arnaud-moncel) [inline test-expected/sequelize/dumper-output/index.expected.js:40]:
```suggestion
        const model = connection.import(path.join(__dirname, modelsDir, file));
```

**Review by arnaud-moncel — COMMENTED** [review]
Two small forgets on the generation of sequelize `models/index`.
One thing, I think we can for the case of only one database return the connection 
```js
if(databasesConfiguration.length === 1) {
  db.sequelize = Object.values(connections)[0];
//or
  db.mongoose = Object.values(connections)[0];
}
```
I'm know is not very beauty, but i dont have any idea now.

File: templates/app/models/index.hbs [hunk 3]
```diff
@@ -1,97 +1,98 @@
...
+    databaseOptions.dialectOptions.options = { encrypt: true };
+    {{else}}
+    const rejectUnauthorized = process.env.DATABASE_REJECT_UNAUTHORIZED;
+    if (rejectUnauthorized && (JSON.parse(rejectUnauthorized.toLowerCase()) === false)) {
+      databaseOptions.dialectOptions.ssl = { rejectUnauthorized: false };
+    } else {
+      databaseOptions.dialectOptions.ssl = true;
     }
-  });
+{{/if}}
```
Comment (jeffladiray) [inline templates/app/models/index.hbs:54]: [Bug] Appear to be a bug in the handlebars library. Maintaining correct indentation here breaks the generation

Comment (arnaud-moncel) [reply templates/app/models/index.hbs:54]: https://github.com/handlebars-lang/handlebars.js/issues/1716

**Review by arnaud-moncel — APPROVED** [review]
lGTM ✨

**Review by arnaud-moncel — COMMENTED** [review]
Code approved missing quick manual tests

**Review by arnaud-moncel — COMMENTED** [review]
Code after rebase seem to be ok

**Review by arnaud-moncel — APPROVED** [review]
NIIIIIICE 👌

**Pull request merged**